.github/pull_request_template.md

```diff
@@ -6,12 +6,15 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
-
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+If the provided content is part of the present PR remove the # symbol.
+
+- [ ] **changelog** provided | not required because: 
+- [ ] **development PR** provided # | not required because: 
+- [ ] **framework PR** provided # | not required because: 
+- [ ] **3.6 PR** provided # | not required because: 
+- [ ] **2.28 PR** provided # | not required because: 
+- **tests**  provided | covered # | not required because: 
```

Comment:
Sometimes there is testing that can't be automated. As a reviewer and gatekeeper, I want to see that in a standard place in the pull request description, so that there's no risk of missing it when it's buried in a long description.
```suggestion
- **tests**  provided | not required because: | additional check that can't be automated: 
```

Reply:
As above I will not be adding this section, since it can be covered in the justification section.

- **tests**  not required because:  additional check that can't be automated.

Reply:
There are sometimes additional checks but also some tests are provided. But ok, that's not a blocker, let's see how it goes.